libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestClient.java

```diff
@@ -363,6 +370,44 @@ public URI resolveUrl(String path) {
 		}
 	}
 
+    /**
+     * Use a unauthenticated client info when do not need authentication support (e.g.
+     * if you are talking to non-salesforce servers)
+     *
+     * NB: Your RestRequest's path will need to be a complete URL
+     */
+    public static class UnauthenticatedClientInfo extends ClientInfo {
```

Comment:
You will get a different instance with every call but internally it will leverage the same volley RequestQueue